css/sudoku.css

```
@@ -0,0 +1,215 @@
+}
+
+.sdk-col.sdk-selected {
+	background-color: rgba(100,100,100,0.2) !important;
+	-webkit-animation: .8s .1s pulse infinite linear;
+  	-moz-animation: .8s .1s pulse infinite linear;
+  	-o-animation: .8s .1s pulse infinite linear;
+  	animation: .8s .1s pulse infinite linear;
+}
+
+@-moz-keyframes pulse{
+	40%{ background:rgba(100,100,100,0.04); }
```

Biome (2.1.2)

[error] Unexpected shorthand property background after background-color



(lint/suspicious/noShorthandPropertyOverrides)